### PR TITLE
Chart data files

### PR DESCRIPTION
**Pull request by ralsina**
Add a data_file option to chart shortcode and directive as sort-of-per #3129

### CODE REVIEW

File: docs/manual.rst [hunk 1]
```diff
@@ -1256,13 +1256,39 @@ chart
...
-        {{% raw %}}{{% chart Bar title='Browser usage evolution (in %)' %}}
-        x_labels='["2002","2003","2004","2005","2006","2007","2008","2009","2010","2011","2012"]'%}}
+        {{% raw %}}{{% chart Bar title='Browser usage evolution (in %)'
+x_labels='["2002","2003","2004","2005","2006","2007"]' %}}
         'Firefox', [None, None, 0, 16.6, 25, 31]
         'Chrome',  [None, None, None, None, None, None]
         'IE',      [85.8, 84.6, 84.7, 74.5, 66, 58.6]
         'Others',  [14.2, 15.4, 15.3, 8.9, 9, 10.4]
-        {{% /chart %}}{{% /raw %}}
+        {{% /chart %}}
```
Comment (Kwpolska) [inline docs/manual.rst]: they need to be on the same line

File: docs/manual.rst [hunk 2]
```diff
@@ -1256,13 +1256,39 @@ chart
 
     .. code:: text
 
-        {{% raw %}}{{% chart Bar title='Browser usage evolution (in %)' %}}
-        x_labels='["2002","2003","2004","2005","2006","2007","2008","2009","2010","2011","2012"]'%}}
+        {{% raw %}}{{% chart Bar title='Browser usage evolution (in %)'
+x_labels='["2002","2003","2004","2005","2006","2007"]' %}}
```
Comment (Kwpolska) [inline docs/manual.rst]: Is this a syntax change or a fix to the docs?

File: nikola/plugins/shortcode/chart.py [hunk 3]
```diff
@@ -50,20 +50,31 @@ def handler(self, chart_type, **_options):
...
+        style_name = options.pop('style', 'BlueStyle')
         if '(' in style_name:  # Parametric style
             style = eval('pygal.style.' + style_name)
         else:
             style = getattr(pygal.style, style_name)
-        for k, v in _options.items():
-            try:
-                options[k] = literal_eval(v)
+        for k, v in options.items():
+            try:                options[k] = literal_eval(v)
```
Comment (Kwpolska) [inline nikola/plugins/shortcode/chart.py]: why is this on one line?

File: docs/manual.rst (hunk 2 above)
Comment (ralsina) [reply docs/manual.rst]: Just a fix for the docs, things were different lengths

File: docs/manual.rst (hunk 1 above)
Comment (ralsina) [reply docs/manual.rst]: Sigh. Fixing.

File: docs/manual.rst (hunk 2 above)
Comment (Kwpolska) [reply docs/manual.rst]: You moved x_labels from “inside” the shortcode to its arguments.

Comment (ralsina) [reply docs/manual.rst]: Hmmm that was broken, I think.

**Pull request merged**